src/devices/tuya.ts

```diff
@@ -20371,5 +20371,55 @@ export const definitions: DefinitionWithExtend[] = [
                 [103, "buzzer_mute", tuya.valueConverter.onOff],
             ],
         },
+        {
+        fingerprint: tuya.fingerprint("TS030F", ["_TZ3210_sxtfesc6"]),
+        model: "ADCBZI01",
+        vendor: "Moes",
+        description: "Curtain Robot",
+        fromZigbee: [fz.cover_position_tilt, tuya.fz.datapoints],
+        toZigbee: [tz.cover_position_tilt, tz.cover_state, tuya.tz.datapoints],
```

Comment:
```suggestion
        toZigbee: [tuya.tz.datapoints],
```

does it also work with this and without `fz.cover_position_tilt,`?